app/_src/policies/meshtcproute.md

```diff
@@ -0,0 +1,256 @@
+
+For more information, see the [matching docs](/docs/{{ page.version }}/policies/targetref).
+
+## Configuration
+
+Unlike other outbound policies, `MeshTCPRoute` doesn't contain `default`
+directly in the `to` array. The `default` section is nested inside `rules`,
+so the policy structure looks like the following:
+
+```yaml
+spec:
+  targetRef: # top-level targetRef selects a group of proxies to configure
+    kind: Mesh|MeshSubset|MeshService|MeshServiceSubset 
+  to:
+    - targetRef: # targetRef selects a destination (outbound listener)
+        kind: MeshService
+        name: backend
+      rules:
+        - default: # configuration applied for the matched TCP traffic
+            backendRefs: [...]
+```
+
+### Default configuration
+
+The following describes the default configuration settings of the `MeshTCPRoute` policy:
+
+- **`backendRefs`**: (Optional) List of destinations for the request to be redirected to
+  - **`kind`**: Either `MeshService` or `MeshServiceSubset`
+  - **`name`**: The service name
+  - **`tags`**: Service tags. These must be specified if the `kind` is 
+    `MeshServiceSubset`.
+  - **`weight`**: When a request matches the route, the choice of an upstream
+    cluster is determined by its weight. Total weight is a sum of all weights
+    in the `backendRefs` list.
+
+
+## Examples
+
+### Traffic split
+
+You can use `MeshTCPRoute` to split TCP traffic between services with
+different tags and implement A/B testing or canary deployments.
+
+Here's an example of a `MeshTCPRoute` that splits the traffic from 
+`frontend_kuma-demo_svc_8080` to `backend_kuma-demo_svc_3001` between versions:
+
+{% tabs split useUrlFragment=false %}
+{% tab split Kubernetes %}
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshTCPRoute
+metadata:
+  name: tcp-route-1
+  namespace: {{site.mesh_namespace}}
+  labels:
+    kuma.io/mesh: default
+spec:
+  targetRef:
+    kind: MeshService
+    name: frontend_kuma-demo_svc_8080
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend_kuma-demo_svc_3001
+      rules:
+        - default:
+            backendRefs:
+              - kind: MeshServiceSubset
+                name: backend_kuma-demo_svc_3001
+                tags:
+                  version: "1.0"
+                weight: 90
+              - kind: MeshServiceSubset
+                name: backend_kuma-demo_svc_3001
+                tags:
+                  version: "2.0"
+                weight: 10
+```
+
+You can apply the configuration with `kubectl apply -f [..]`.
+{% endtab %}
+
+{% tab split Universal %}
+
+```yaml
+type: MeshTCPRoute
+name: tcp-route-1
+mesh: default
+spec:
+  targetRef:
+    kind: MeshService
+    name: frontend_kuma-demo_svc_8080
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend_kuma-demo_svc_3001
+      rules:
+        - matches:
+            - path:
+                type: PathPrefix
+                value: /api
+          default:
```

Comment:
this part is missing from k8s equivalent so I'd remove that

```suggestion
        - default:
```

Reply:
yeah it's definitely not part of MeshTCPRoute